SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf

```diff
@@ -502,6 +517,14 @@ Consensus:Review.</skos:editorialNote>
 		<rdfs:label xml:lang="en">private label m b s deal</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;RealEstateMortgageInvestmentConduit">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-fund-fund;SpecialPurposeVehicle"/>
+		<rdfs:label xml:lang="en">real estate mortgage investment conduit</rdfs:label>
+		<skos:definition xml:lang="en">special purpose vehicle or debt instrument that pools mortgage loans together and issues mortgage-backed securities</skos:definition>
```

Comment:
No, it must be either an instrument or a legal entity (SPV)

Reply:
@rivettp We made this change during our FCT discussion